cmd/createtree/pkcs11_test.go

```diff
@@ -0,0 +1,71 @@
+// +build pkcs11
+
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+func TestRunPkcs11(t *testing.T) {
+	err := os.Chdir("../..")
```

Comment:
The testdata/pkcs11-conf.json config file contains a file path that is relative to the project root dir, so the test needs to change the working directory.